demisto_sdk/commands/format/update_playbook.py

```diff
@@ -152,6 +152,16 @@ def update_playbook_task_name(self):
                 if task_name:
                     task['task']['name'] = task_name
 
+    def remove_empty_fields_set_incident(self):
+        """Removes unnecessary empty fields from setIncident script"""
+        for task_id, task in self.data.get('tasks', {}).items():
+            script = task.get('task', {}).get('script', '')
+            if 'setIncident' in script:
```

Comment:
let's filter the inputs of `setIndicator`, `createNewIncident` and `createNewIndicator` too

Reply:
Done